### ISSUE TITLE
How can I run in Xcode ?

### ISSUE THREAD

**Issue by XcqRomance** [mull-project/llvm-jit-objc#10]
HI, stanislaw, How do I configure it? I clone the source and I want to generate a xcode project in llvm-jit-objc, what's the confuration step. I'm interesting in execute Objective-C code via LLVM JIT

**Comment by stanislaw** [comment]
Hi @XcqRomance,

Thank you for your interest in this project!

I have cleaned the project up a bit and created initial instructions in the README file.

Please let me know if you can create an Xcode project based on the instructions. I am happy to help you to get it running and also give you some "getting started" advice.

**Comment by XcqRomance** [comment]
Hi @stanislaw ,
Thank you very much for your initial instructions in the README file. But I encounter an error in 'Generating fixtures' step with 'cd fixtures && make all' , 
## the error as follows: 
```
cd /opt/llvm-jit-objc/fixtures/bitcode && \
	  /opt/llvm-6.0.0/bin/clang -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -F/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks -fobjc-arc -emit-llvm /opt/llvm-jit-objc/fixtures/xctest_objc_001_minimal_xctestcase_run/xctest_objc_001_minimal_xctestcase_run.m -c -o /opt/llvm-jit-objc/fixtures/bitcode/xctest_objc_001_minimal_xctestcase_run.bc
In file included from /opt/llvm-jit-objc/fixtures/xctest_objc_001_minimal_xctestcase_run/xctest_objc_001_minimal_xctestcase_run.m:1:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks/XCTest.framework/Headers/XCTest.h:63:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks/XCTest.framework/Headers/XCUIApplication.h:6:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks/XCTest.framework/Headers/XCUIElement.h:12:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/AppKit.framework/Headers/AppKit.h:12:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/AppKit.framework/Headers/AppKitErrors.h:10:1: error: 
      expected '('
API_UNAVAILABLE_BEGIN(ios)
^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/usr/include/os/availability.h:104:40: note: 
      expanded from macro 'API_UNAVAILABLE_BEGIN'
    #define API_UNAVAILABLE_BEGIN(...) _Pragma("clang attribute push") __API_UNAVAILABLE_BEGIN_GET_M...
                                       ^
<scratch space>:84:22: note: expanded from here
 clang attribute push
                     ^
In file included from /opt/llvm-jit-objc/fixtures/xctest_objc_001_minimal_xctestcase_run/xctest_objc_001_minimal_xctestcase_run.m:1:
```
## the envirenment of my mac as follows: 
macOS Catalina 10.15, Xcode version: 11.1, binary version of LLVM is 6.0.

How can resolve this error and generate a Xcode project ? I'm looking forward to you reply. Thanks !

**Comment by XcqRomance** [comment]
hi @stanislaw I resolved the this error，I downgrade the Xcode 11.1 to Xcode 10.3, and the generate Xcode project successfully. thank you.

**Comment by stanislaw** [comment]
@XcqRomance I don't have 10.15 and 11.1 but I would assume there should be some typical workaround. Usually the changes needed for such upgrades are more or less google-able.
